addons/survey/static/src/scss/survey_templates_results.scss

```diff
@@ -9,9 +9,16 @@
     .tab-content > .tab-pane {
         display: block;
     }
+    .toPrint {
+        white-space: pre-wrap;
+    }
     html {
         height: unset;
     }
+    @page {
```

Comment:
Hello :wave: 

So, after double-checking, it seems that these css rules are global and loaded into the backend, which is less than great.

I opened a task inside our team to take a look:
https://www.odoo.com/web#cids=1&menu_id=6444&action=5050&model=project.task&view_type=form&id=3666858

We will probably have to wait for that to be fixed before fixing this specific use case here, to avoid undesired side-effects.

Reply:
Hello @awa-odoo 

Any updates on this?

Reply:
No news yet, I'll try to make things move along.

Reply:
Hello @awa-odoo 

I see that the other PR linked to the task above has been deployed. So do you have any updates regarding my fix? :slightly_smiling_face:

Reply:
Hello @yoba-odoo 

Thanks for the reminder!
Indeed the related PR is now merged, which means we should be able to move along with this one, sorry for the delay.

A few things to check after re-reading this:
- This CSS is applied to all survey templates, meaning it also applies to the survey statistics page for example, could you check that it does not negatively impact them, notably by the usage of the margin: 0 rule?
- It would be nice to find a solution that does not alter the XML template, as it can create issues for clients that have custom code, especially in such old stables, can't we instead add some printing style on the "o_survey_question_text_box" class to achieve the same result?

Cheers!

Reply:
hi,

> can't we instead add some printing style on the "o_survey_question_text_box" class to achieve the same result?

maybe with "[field-sizing: content](https://developer.mozilla.org/en-US/docs/Web/CSS/field-sizing);    resize: none;" but it's rather new (from [mdn page](https://developer.mozilla.org/en-US/docs/Web/CSS/field-sizing) it doesn't work in firefox/safari)"

otherwise we could also have the current solution and make it less XPATH dangerous in stable by keeping the old code but adding `"t-if="False"` and adding the new code after

Reply:
That seems rather dirty :smile: 

Note that I would be less opposed to a solution involving beforeprint / afterprint in JS, as it adds no risks for stable. If we can find something concise and clean that works well.